doc/content/guides/connecting-gateways/ciscoixm/_index.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Cisco LoRaWAN Gateway"
+description: ""
+weight: 1
+---
+
+The Cisco LoRaWAN Gateway is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.cisco.com/c/en/us/products/routers/wireless-gateway-lorawan/). This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. Cisco LoRaWAN Gateway with latest firmware version `2.0.32`.
+3. A console cable from USB to RJ45.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}).  
+The **EUI** is derived from the **MAC_ADDRESS** that can be found on the back panel of the gateway. To get the **EUI** from the **MAC_ADDRESS** insert FFFE after the first 6 characters to make it a 64bit EUI.
+
+If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) the **Gateway Server Address** is the same as what you use instead of `thethings.example.com`.
+
+## Configuration
+  
+Plug the RJ45 end of the cable in the Console port of the gateway, and the USB port to your computer.  
+Then, if you are using MacOS or Linux, connect to the Gateway by opening a terminal and a executing the following commands.  
+Use PuTTy if you are using Windows.
```

Comment:
Same about line breaks, instructions and links. Please see other guides and the documentation guidelines in `CONTRIBUTING.md`